### ISSUE TITLE
System.stacktrace is deprecated

### ISSUE THREAD

**Issue by IceDragon200** [elixir-ecto/connection#12]
And has been deprecated for some time now

```
==> connection
Compiling 1 file (.ex)
warning: System.stacktrace/0 is deprecated, use __STACKTRACE__ instead
  lib/connection.ex:429

warning: System.stacktrace/0 is deprecated, use __STACKTRACE__ instead
  lib/connection.ex:431

warning: System.stacktrace/0 is deprecated, use __STACKTRACE__ instead
  lib/connection.ex:491

warning: System.stacktrace/0 is deprecated, use __STACKTRACE__ instead
  lib/connection.ex:540

warning: System.stacktrace/0 is deprecated, use __STACKTRACE__ instead
  lib/connection.ex:581

warning: System.stacktrace/0 is deprecated, use __STACKTRACE__ instead
  lib/connection.ex:625

warning: System.stacktrace/0 is deprecated, use __STACKTRACE__ instead
  lib/connection.ex:628

warning: System.stacktrace/0 is deprecated, use __STACKTRACE__ instead
  lib/connection.ex:631

warning: System.stacktrace/0 is deprecated, use __STACKTRACE__ instead
  lib/connection.ex:657

warning: System.stacktrace/0 is deprecated, use __STACKTRACE__ instead
  lib/connection.ex:660

warning: System.stacktrace/0 is deprecated, use __STACKTRACE__ instead
  lib/connection.ex:663

warning: System.stacktrace/0 is deprecated, use __STACKTRACE__ instead
  lib/connection.ex:744

warning: System.stacktrace/0 is deprecated, use __STACKTRACE__ instead
  lib/connection.ex:770

warning: System.stacktrace/0 is deprecated, use __STACKTRACE__ instead
  lib/connection.ex:813
```

**Comment by altdsoy** [comment]
I wanted to say by linking the following PR that it was already fixed and we just need to wait for the merge and for the release...
Only to notice that it's was you :+1: 
Still linking for Github notification..
https://github.com/fishcakez/connection/pull/13

**Comment by ericmj** [comment]
Fixed in #13.
